lmnet/tox.ini

```diff
@@ -7,7 +7,7 @@ skipsdist = True
 [testenv]
 recreate = True
 deps =
-    -r{toxinidir}/gpu.requirements.txt
+    -r{toxinidir}/../gpu.requirements.txt
```

Comment:
@suttang @hadusam 
In docker image, we have requreiments.txt in `/tmp/requirements/`, When tox executed, these files cannot find requirements.txt. Do you have any idea to find path in tox using docker?

Reply:
IMO tox command via docker will not be used much.
If you want to run via docker, you just use -v.

Reply:
>  tox command via docker will not be used much.

`make test-lmnet` use tox. So this line cause CI failure.
Thinking time...